pkg/plugins/golang/v3/commons.go

```diff
@@ -78,7 +78,7 @@ manifests: controller-gen`
 		}
 
 		if err := util.ReplaceInFile("Makefile",
-			"ENVTEST_K8S_VERSION = 1.23",
+			"ENVTEST_K8S_VERSION = 1.23.3",
```

Comment:
this change will also change the plugin scaffolds so we need to do in the follow up,
See that it is failing in the test data generate check: https://github.com/kubernetes-sigs/kubebuilder/runs/5760100478?check_suite_focus=true#step:6:895

Reply:
Sorry, by mistake.